### DISCUSSION TITLE
Ability to change Window Titles

### DISCUSSION THREAD

**Discussion by mem16421** [isontheline/pro.webssh.net#459] [Ideas]
When showing multiple open connections, it lists them by Window 1, Window 2, etc…. There should be a way to rename them, or allow a title to be set from the connection profile

**Comment by isontheline** [comment]
Hello @mem16421 👋

Thank you a lot for your feedback!

You're right the titles should be better than Window X.
I'm opening an issue with your request : https://github.com/isontheline/pro.webssh.net/issues/460

Have a nice day ☀️
